### ISSUE TITLE
Can we have slots for .vuecal__now-line in cell

### ISSUE THREAD

**Issue by Vineet2611** [antoniandre/vue-cal-v4#305]
Current time in scheduler depends upon system time but I have a use case where user timezone is different than the system in which app is running. So can we have some provision to alter the now timezone either from props or from slots ? 

Thanks :)

**Comment by wolph** [comment]
I have a use-case for this as well. Did you find a solution @Vineet2611 ?

**Comment by Vineet2611** [comment]
Yes... I would say it is a workaround rather than a proper solution. 2 things I did on  '@ready' hook to achieve this

1. Changed the 'now' property of 'vuecal' as per the required timezone and kept it intact during traversal because when we navigate 'now' gets refreshed and takes the current date of browser. 

2. Changed the position of current time pivot as per the new 'now' by DOM manipulation.

By doing this my timeframe reference of vuecal canvas changed to the required timezone. But it could have been more convenient if we had external control to handle 'now' property.

Hope this will help... :)
